server/services/questionsToDelete.ts

```diff
@@ -0,0 +1,115 @@
+import { findAnswerConfigByCode, IncidentTypeConfiguration } from '../data/incidentTypeConfiguration/types'
+
+/**
+ * This interface is a subset of the API's `Question`
+ *
+ * The logic to determine questions to delete only needs
+ * the question `code` and the responses' `response`
+ */
+interface AnsweredQuestion {
+  code: string
+  responses: { response: string }[]
+}
+
+export default class QuestionsToDelete {
```

Comment:
not really a big deal, but why a class with only static methods? there's no share state?

Reply:
At some point I almost had the config as state but I removed. I still prefer this to the alternative (a bunch of functions) as these methods are all related to each other.

Reply:
well a file with 1 public and 2 private methods achieves the same these-are-related aim. no, i'm not demanding a change here ;)
but i was surprised to see that the `this` keyword works in a static context

Reply:
We shouldn't be afraid of using classes to group things ;) - In my mind determine which questions to delete is providing that specific service and the traversal, etc. stuff is internal implementation that the client doesn't care/need to know ☺️